cmake/FindHerbert.cmake

```diff
@@ -37,6 +37,10 @@ set(DIRS_TO_SEARCH
     ENV{ProgramFiles}/Herbert/herbert_core
 )
 
+if(Herbert_ROOT AND NOT EXISTS "${Herbert_ROOT}/herbert_init.m")
```

Comment:
what is the purpose of this check? The next command is a `find_path`

Reply:
See commit message of https://github.com/pace-neutrons/Horace/pull/147/commits/678850436e6ca324ea2cfbd76b6279d6af224804. If `Herbert_ROOT` is passed to CMake on the command line, `find_path` won't do anything because `Herbert_ROOT` is already a cache variable.

Reply:
Offline discussion: the `find_path` call is to be moved into an `else()` block

Reply:
It would remove the ambiguity to move the following `find_path(Herbert_ROOT, ...` into an `else()` block.

If it's defined and valid - the `find_path` will execute and short-circuit
if it's defined and invalid - the new if() check will catch an raise a FATAL_ERROR
if it's undefined - the `find_path` will execute and (hopefully) find it.